exporter/keystoneexporter/keystone.go

```diff
@@ -64,12 +64,22 @@ type KsMessage struct {
 }
 
 type KsPayload struct {
-	Ec2InstanceId string           `json:"ec2_instance_id"`
-	NflxStack     string           `json:"stack"`
-	Metric        metricspb.Metric `json:"metric"`
+	Ec2InstanceId string            `json:"ec2_instance_id"`
+	NflxStack     string            `json:"stack"`
```

Comment:
I think it would be good to rename this, for clarity, to `ec2_stack` or similar.

Reply:
Renamed to `nf_stack`.  Comes form env var `NETFLIX_STACK`